misc/packaging/README.md

```diff
@@ -0,0 +1,117 @@
+# ubuntuSoS
+this scripts show how to build acrn-hypervisor and acrn-kernel deb package
+
+1. setup native ubuntu
+
+install python3
+
+config proxy
+
+sudo su
+
+
+2. config json file
+
+
+{
+
+	"//":"release ubuntu as sos verion",
+
+	"install_package":"true",---------->this 1st time should set to true if you did not install acrn related compile package, if already install,set to false
+
+	"gcc_version":"7.3.0",---------->higher than this
+
+	"binutils":"2.27",---------->higher than this
+
+	"//":"acrn-hypervisor config",
+
+	"build_acrn":"true",---------->if you need build acrn-hypervisor
+
+	"sync_acrn_code":"true",---------->if you need sync acrn-hypervisor code 
```

Comment:
How do you know if this is needed?

Reply:
user should know whether to sync code or not